### PR TITLE
Help Them Keep Learning

### PR DESCRIPTION
**Pull request by yokolet**
I finally finished the document, "Help them keep learning". Please have a look and merge this.

I really appreciate if someone of San Francisco chapter adds the info about after workshop events.

### CODE REVIEW

**Comment by seancorfield** [comment]
I think this is a great starting point for this! Thank you @yokolet ! :+1:

File: Help-Them-Keep-Learning.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Follow-up meetups in the past
+
+1. Durham NC - Intro to Clojure at a local Clojure meetup
+2. San Francisco - 
+
+
+### ClojureBridge Alumni mailing list
+
+[https://groups.google.com/forum/#!forum/clojurebridge-alumni](https://groups.google.com/forum/#!forum/clojurebridge-alumni)
```
Comment (davidchambers) [inline Help-Them-Keep-Learning.md]: In Markdown one can use angle brackets for a link if using the URL as the link text. In this case:

```diff
-[https://groups.google.com/forum/#!forum/clojurebridge-alumni](https://groups.google.com/forum/#!forum/clojurebridge-alumni)
+<https://groups.google.com/forum/#!forum/clojurebridge-alumni>
```

**Comment by davidchambers** [comment]
:+1:

**Comment by yokolet** [comment]
@davidchambers Thanks! I changed links to use angle brackets.

**Comment by kpfell** [comment]
I'm :+1:ing this, as well--and now there are three :+1:s! So I'm merging it. :wink:

**Pull request merged**